source/simple_socketio.ts

```diff
@@ -55,6 +55,7 @@ export default class SimpleSocketIOClient {
   private packetQueue: string[] = [];
   private reconnectionAttempts: number = 0;
   private reconnecting: boolean = false;
+  initializingPromise: Promise<void>;
```

Comment:
shouldn't this be private?

Reply:
ah, i see you're using it in the test. not a fan of publicly exposing internal state for use in tests, but the tests are already pretty low-level, so I guess this will do